packages/fast-css-editor-react/src/editor.props.ts

```diff
@@ -0,0 +1,12 @@
+import { ManagedClasses } from "@microsoft/fast-jss-manager";
+import { CSSEditorClassNameContract } from "./editor.style";
+import { CSSPositionHandledProps } from "./";
+
+export interface CSSEditorUnhandledProps extends React.HTMLAttributes<HTMLDivElement> {}
+
+export type CSSEditorComponentsHandledProps = CSSPositionHandledProps &
```

Comment:
do you actually need this type definition? Can't you just define `CSSEditorHandledProps` as these two types?

Reply:
this is going to be extended for every component that is added to the editor, so for example one we will be adding after this will be CSSMarginHandledProps.